src/clj/puppetlabs/services/file_serving/config/puppet_fileserver_config_core.clj

```diff
@@ -0,0 +1,93 @@
+(ns ^{:doc
+      "Tools to parse Puppet fileserver.conf."}
+    puppetlabs.services.file-serving.config.puppet-fileserver-config-core
+  (:require
+    [puppetlabs.kitchensink.core :as ks]
+    [schema.core :as schema]
+    [puppetlabs.puppetserver.ringutils :as ring])
+  (:use [clojure.java.io :only (reader)]
```

Comment:
Yup, we tend to avoid `:use` at all costs, always use `:require mynamespace :refer [function1 function2]`, and sometimes also `:refer :all` for cases like testing or DSL usage.